common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -74,6 +81,53 @@ protected void throwIfNetworkNotAvailable(final Context context) throws ClientEx
         );
     }
 
+
+    protected AuthorizationRequest getAuthorizationRequest(final OAuth2Strategy strategy,
```

Comment:
Both input parameters should be "notnull", right? We need to have a null check here.